src/controllers/dynakube/activegate/reconciler_test.go

```diff
@@ -192,3 +204,149 @@ func getTestActiveGateService(t *testing.T, fakeClient client.Client) corev1.Ser
 
 	return activegateService
 }
+
+func TestExclusiveSynMonitoring(t *testing.T) {
+	dtRequests := &dtclient.MockDynatraceClient{}
+	dtRequests.On(
+		"GetActiveGateAuthToken",
+		testName,
+	).Return(
+		&dtclient.ActiveGateAuthTokenInfo{},
+		nil)
+	dynaKube := &dynatracev1beta1.DynaKube{
+		ObjectMeta: syntheticCapabilityObjectMeta,
+	}
+	k8sRequests := fake.NewClient(testKubeSystemNamespace)
+	reconciler := NewReconciler(
+		context.TODO(),
+		k8sRequests,
+		k8sRequests,
+		scheme.Scheme,
+		dynaKube,
+		dtRequests)
+	err := reconciler.Reconcile()
+
+	toAssertReconciliation := func(t *testing.T) {
+		require.NoError(t, err, "successfully reconciled for syn-mon")
+	}
+	t.Run("for-errorless-reconciliation", toAssertReconciliation)
```

Comment:
use `t.Run()` directly, instead of creating the test func before the test run

Reply:
Reworded.